config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  root 'works#main'
+  resources :works
+    post 'works/:id/upvote', to: 'works#upvote', as: 'upvote'
```

Comment:
The indentation doesn't actually mean anything here as it's not technically nested inside.